scripts/examples/.env.bootnode.example

```diff
@@ -5,3 +5,6 @@ BOOTNODES=enode://e21e2053005e40653d055fc01a07768357749d27d630702c203b1a3c00bdf2
 
 # in case `sudo` is needed
 PERMISSION_PREFIX=
+
+# Use fuses pre-exsisting snapshot to speed up initial sync (note this will require additional disk space to store the temp zip)
+USE_SNAPSHOT=0
```

Comment:
can you change this to true/false?

Reply:
@leonprou done sorry for the delay here. This should be good to go now :)